bgpd/bgpd.c

```diff
@@ -2621,6 +2630,9 @@ int peer_group_remote_as_delete(struct peer_group *group)
 	for (ALL_LIST_ELEMENTS(group->peer, node, nnode, peer)) {
 		other = peer->doppelganger;
 
+		if (CHECK_FLAG(peer->flags, PEER_FLAG_CAPABILITY_ENHE))
```

Comment:
Same comment as above.

Reply:
Same answer as above.  When peer-group remote-as command is deleted, all of the peers under that peer-group are deleted.